src/domain/Session.js

```diff
@@ -71,6 +71,13 @@ export default class Session {
     return this.profile ? `${this.profile.firstName} ${this.profile.lastName}` : '';
   }
 
+  hasAccessToVoicemail(): boolean {
+    if (!this.profile) {
```

Comment:
`return !this.profile ? false : !!this.profile.voicemail;`

Reply:
For some reasons Flow blocked me.